### ISSUE TITLE
Eventchannel returns a different value for some fields

### ISSUE THREAD

**Issue by cristgl** [wazuh/wazuh#2937]
|Wazuh version|Install type|Install method|Platform|
|---|---|---|---|
| 3.9.0-3909 | Manager | Packages/Sources | Windows/Linux |

When collecting Windows events, some fields are retrieved with an ID and not their exact value, for example:

![imagen](https://user-images.githubusercontent.com/14914658/55065682-948dd280-507c-11e9-83ca-76c753f0df0b.png)

The category field is not returned as 'Logon/Logoff', instead, we got a field named `categoryId` with value `%%8273`.

```
{"win":{"system":{"providerName":"Microsoft-Windows-Security-Auditing","providerGuid":"{54849625-5478-4994-A5BA-3E3B0328C30D}","eventID":"4719","version":"0","level":"0","task":"13568","opcode":"0","keywords":"0x8020000000000000","systemTime":"2019-03-27T09:38:32.465664100Z","eventRecordID":"15711103","processID":"564","threadID":"3036","channel":"Security","computer":"pcname","severityValue":"AUDIT_SUCCESS","message":"System audit policy was changed."},"eventdata":{"subjectUserSid":"S-1-5-18","subjectUserName":"PCNAME$","subjectDomainName":"WORKGROUP","subjectLogonId":"0x3e7","categoryId":"%%8273","subcategoryId":"%%12545","subcategoryGuid":"{0CCE9216-69AE-11D9-BED3-505054503030}","auditPolicyChanges":"%%8451"}}}
```

**Comment by Skeptor** [comment]
Testing of the solution:

Memory Report
-------------------------
Execution of valgrind in `ossec-analaysisd`
```
==16612== LEAK SUMMARY:
==16612==    definitely lost: 100 bytes in 9 blocks
==16612==    indirectly lost: 0 bytes in 0 blocks
==16612==      possibly lost: 28,832 bytes in 106 blocks
==16612==    still reachable: 11,997,490 bytes in 56,930 blocks
==16612==         suppressed: 0 bytes in 0 blocks
==16612== Reachable blocks (those to which a pointer was found) are not shown.
==16612== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==16612== 
```

Memory used of the ossec-agent in Windows. The powershell script that I use is this:
`For (0=0; 0 -le 1000000; 0++) {Get-Process -Name ossec-agent | Select-Object Name,@{Name='WorkingSet';Expression={(.WorkingSet)}}; sleep 2}`
```
ossec-agent                                                            14094336
ossec-agent                                                            14094336
ossec-agent                                                            14094336
ossec-agent                                                            14094336
ossec-agent                                                            14123008
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14172160
ossec-agent                                                            14393344
ossec-agent                                                            14393344
ossec-agent                                                            14393344
ossec-agent                                                            14569472
ossec-agent                                                            14569472
ossec-agent                                                            14725120
ossec-agent                                                            14725120
ossec-agent                                                            14725120
ossec-agent                                                            14888960
ossec-agent                                                            14934016
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15073280
ossec-agent                                                            15134720
ossec-agent                                                            15134720
ossec-agent                                                            15134720
ossec-agent                                                            15134720
ossec-agent                                                            15134720
ossec-agent                                                            15134720
ossec-agent                                                            15130624
ossec-agent                                                            15130624
ossec-agent                                                            15130624
ossec-agent                                                            15126528
ossec-agent                                                            15126528
ossec-agent                                                            15118336
ossec-agent                                                            15142912
ossec-agent                                                            15142912
ossec-agent                                                            15138816
ossec-agent                                                            15138816
ossec-agent                                                            15138816
ossec-agent                                                            15138816
ossec-agent                                                            15138816
ossec-agent                                                            15138816
ossec-agent                                                            15138816
ossec-agent                                                            15020032
ossec-agent                                                            15020032
ossec-agent                                                            15020032
ossec-agent                                                            15020032
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15179776
ossec-agent                                                            15179776
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15151104
ossec-agent                                                            15147008
ossec-agent                                                            15147008
ossec-agent                                                            15147008
ossec-agent                                                            15147008
ossec-agent                                                            15147008
ossec-agent                                                            15147008
ossec-agent                                                            15147008
ossec-agent                                                            15290368
ossec-agent                                                            15261696
ossec-agent                                                            15261696
ossec-agent                                                            15261696
ossec-agent                                                            15261696
ossec-agent                                                            15261696
ossec-agent                                                            15261696
ossec-agent                                                            15597568
ossec-agent                                                            15597568
ossec-agent                                                            15597568
ossec-agent                                                            15597568
ossec-agent                                                            15613952
ossec-agent                                                            15613952
ossec-agent                                                            15757312
ossec-agent                                                            15757312
ossec-agent                                                            15757312
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15925248
ossec-agent                                                            15896576
ossec-agent                                                            15896576
ossec-agent                                                            15921152
ossec-agent                                                            15921152
ossec-agent                                                            16003072
ossec-agent                                                            16003072
ossec-agent                                                            16003072
ossec-agent                                                            16003072
ossec-agent                                                            16003072
ossec-agent                                                            16003072
ossec-agent                                                            16003072
ossec-agent                                                            15912960
ossec-agent                                                            15912960
ossec-agent                                                            16027648
ossec-agent                                                            16531456
ossec-agent                                                            16531456
ossec-agent                                                            16531456
ossec-agent                                                            16531456
ossec-agent                                                            16605184
ossec-agent                                                            16605184
ossec-agent                                                            16576512
ossec-agent                                                            16883712
ossec-agent                                                            16883712
ossec-agent                                                            17051648
ossec-agent                                                            17051648
ossec-agent                                                            17207296
ossec-agent                                                            17362944
ossec-agent                                                            17362944
ossec-agent                                                            17362944
ossec-agent                                                            17526784
ossec-agent                                                            17526784
ossec-agent                                                            17526784
ossec-agent                                                            17526784
ossec-agent                                                            17526784
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17768448
ossec-agent                                                            17911808
ossec-agent                                                            17911808
ossec-agent                                                            18067456
ossec-agent                                                            18231296
ossec-agent                                                            18452480
ossec-agent                                                            18604032
ossec-agent                                                            18849792
ossec-agent                                                            19062784
ossec-agent                                                            19062784
ossec-agent                                                            19218432
ossec-agent                                                            19218432
ossec-agent                                                            19218432
ossec-agent                                                            19439616
ossec-agent                                                            19705856
ossec-agent                                                            19705856
ossec-agent                                                            19943424
ossec-agent                                                            19943424
ossec-agent                                                            20008960
ossec-agent                                                            20209664
ossec-agent                                                            20209664
ossec-agent                                                            20451328
ossec-agent                                                            20451328
ossec-agent                                                            20451328
ossec-agent                                                            20451328
ossec-agent                                                            20451328
```
The memory of the process increases slowly on the normal behaviour but each audit change increase the memory used in ~200KB.

The result on the memory test is cropped, after a short execution of the agent the memory increases fast and then decrease till it reach 12 MB. Still I have to run Dr. Memory to check if the increase is a Memory Leak or it remains until the agent cleans the queue. 

Failed tests
----------------
The PR only works in English versions of Windows. When Windows is configured in another language the decoder can't find the equivalent in text of the field because the XML of the message is always in english but the message information is in the Windows language.

**Comment by chemamartinez** [comment]
Merged: #2950
